### DISCUSSION TITLE
Continued?

### DISCUSSION THREAD

**Discussion by CuileannA** [Whinarn/UnityMeshSimplifier#60] [Q&A] [answered]
Currently working on a new project using Unity 2021.2.15f1, I'm curious, are there intentions for this asset/package to be maintained or is it an idea no longer being worked on?

**Comment by Whinarn** [comment, marked as answer]
This project is not actively being maintained. There will likely be smaller bugfixes if they are reported. No more features or bigger changes without PRs submitted.

So I wouldn't recommend using this project in a professional capacity. Unless you have resources to maintain it yourself.

Honestly speaking, I have even considered archiving it.
So if there's interest to take over, feel free to reach out to me.
